client/src/panels/result_graphs/result_components.js

```diff
@@ -39,15 +39,18 @@ import { TM, text_maker } from './result_text_provider.js';
 const IndicatorResultDisplay = (props) => {
   const {indicator, is_actual, is_drr17, is_previous} = props;
   if(is_actual){
-    return is_previous ? <span>{indicator_previous_actual_text(indicator)}</span> : <span>{indicator_actual_text(indicator)}</span> ;
+    return is_previous ? <span>{indicator_previous_actual_text(indicator)}</span> : <span>{indicator_actual_text(indicator)}</span>;
   }
+  // vv delete on drr17 exit
   if(!is_drr17){
-    return is_previous ? <span>{indicator_previous_target_text(indicator)}</span> : <span>{indicator_target_text(indicator)}</span> ;
+    return is_previous ? <span>{indicator_previous_target_text(indicator)}</span> : <span>{indicator_target_text(indicator)}</span>;
   }
-  return <span>{drr17_indicator_target_text(indicator)}</span> ;
+  return <span>{drr17_indicator_target_text(indicator)}</span>;
+  // ^^ delete on drr17 exit
```

Comment:
`is_drr17` should be deleted but not just by deleting the text between the comments, which we're likely to do without thinking

Reply:
When I started marking up drr17 specific code, I quickly realized it wouldn't all be a completely straight forward deletion. I kept at it mostly so I kept track of what `drr##` instances I'd dealt with and which I didn't need to bother with.

In all but one or two cases, deleting exactly what I've marked will surface anything complications when the linter complains about undefined/unused variables. I guess I am somewhat relying on me being the one to handle the cleanup as well though. 

Do you think it would be better to take out the deletion flags and leave it as a clean slate for the eventual clean up crew?

Reply:
I dunno. This code in particular won't start throwing errors or linter errors if you just delete that chunk though, it struck me as a particularly dangerous case. At this point it's *probably* stuck enough in both our minds not to be a problem.....hopefully....

Reply:
I could change the comments to `delete, carefully, on drr17 exit` :laughing: